.github/workflows/main.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             ${{ runner.os }}-${{ matrix.python-version }}
 
       - name: Install Deps
-        run: python -m pip install -U tox setuptools virtualenv wheel
+        run: python -m pip install -U tox setuptools virtualenv wheel packaging
       - name: Install and Run Tests
         run: tox -e py
```

Comment:
in this one, I don't really know why we bother installing anything other than `tox`, since it's about to do its own dependency management.  Not important here though.

Reply:
yes, I was also not sure, but thought it wouldn't hurt